Fails with the following error message:

== Sporadic tests ==
date: invalid date ‘+%s’
expr: syntax error: unexpected argument ‘1’
cvs-fast-export: option requires an argument: i
cvs-fast-export: try `cvs-fast-export --help' for more information.
incremental.sh: FAILED
make[1]: *** [/home/builder/aports/testing/cvs-fast-export/src/cvs-fast-export-1.55/tests/Makefile:159: sporadic] Error 1
make: *** [Makefile:147: check] Error 2

diff --git a/tests/Makefile b/tests/Makefile
index c7385cf..690a0c1 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -30,7 +30,7 @@ check: test
 ,v.dot:
 	$(CVS_FAST_EXPORT) -g $< >$*.dot
 
-test: s_regress m_regress r_regress i_regress t_regress c_regress sporadic
+test: s_regress m_regress r_regress i_regress t_regress c_regress # sporadic
 	@echo "No diff output is good news."
 
 rebuild: s_rebuild m_rebuild r_rebuild i_rebuild t_rebuild # z_rebuild
